docs/source/deploying/security.rst

```diff
@@ -52,9 +58,9 @@ You notice that the first line in this snippet references the type "IN_MEMORY".
 
 The last line is the assigment of the authentication system. ::
 
-   servlet.Console.authenticator=inMem
+   servlet.ApiListenerServlet.authenticator=inMem
 
-Here we say that the Console servlet (the Frank!Console) should be protected by the authentication system that we defined before. Without this line, the console remains unprotected. 
+Here we say that the ApiListener servlet should be protected by the authentication system that we defined before. Without this line, the API endpoints remains unprotected. 
 
```

Comment:
Handelt het ApiListener servlet de veiligheid van de Frank!Console af? Kan je hier de vertaalslag maken van de techniek naar de Frank developer of de operator (= gebruiker van Frank!Console)?